chef/roles/provisioner-base.rb

```diff
@@ -2,8 +2,6 @@
 name "provisioner-base"
 description "Provisioner Base role - Apt and Networking"
 run_list(
-         "recipe[provisioner::base]",
-         "recipe[utils]",
-         "recipe[barclamp]"
+         "recipe[provisioner::role_provisioner_base]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.